packages/server/lib/plugins/devserver.js

```diff
@@ -1,141 +1,37 @@
 require('../cwd')
 
-const _ = require('lodash')
 const EE = require('events')
-const path = require('path')
-const debug = require('debug')('cypress:server:preprocessor')
-const Promise = require('bluebird')
-// const appData = require('../util/app_data')
+const debug = require('debug')('cypress:ct:devserver')
 const plugins = require('../plugins')
 
-// const errorMessage = function (err = {}) {
-//   return (err.stack || err.annotated || err.message || err.toString())
-//   .replace(/\n\s*at.*/g, '')
-//   .replace(/From previous event:\n?/g, '')
-// }
-
-// const clientSideError = function (err) {
-//   // eslint-disable-next-line no-console
-//   console.log(err.message)
-//
-//   err = errorMessage(err)
-//
-//   return `\
-// (function () {
-//   Cypress.action("spec:script:error", {
-//     type: "BUNDLE_ERROR",
-//     error: ${JSON.stringify(err)}
-//   })
-// }())\
-// `
-// }
-
 const baseEmitter = new EE()
-let fileObjects = {}
-let fileProcessors = {}
 
 plugins.registerHandler((ipc) => {
-  ipc.on('preprocessor:rerun', (filePath) => {
-    debug('ipc preprocessor:rerun event')
-
-    return baseEmitter.emit('file:updated', filePath)
+  baseEmitter.on('devserver:specs:changed', (specs) => {
+    ipc.send('devserver:specs:changed', specs)
   })
 
-  return baseEmitter.on('close', (filePath) => {
+  return baseEmitter.on('devserver:close', () => {
     debug('base emitter plugin close event')
 
-    return ipc.send('preprocessor:close', filePath)
+    return ipc.send('devserver:close')
   })
 })
 
 // for simpler stubbing from unit tests
 const API = {
-  // errorMessage,
-
-  // clientSideError,
-
   emitter: baseEmitter,
 
-  getFile (filePath, config) {
```

Comment:
deleted a lot of code here - this definitely isnt used anywhere else?

Reply:
No, this was an initial copy paste from preprocessor.js. I created devserver.js in a previous commit.